lib/role.rb

```diff
@@ -50,6 +52,14 @@ module Role
     end
   end
 
+  def billing_administrator?
```

Comment:
If the `billing_administrator` feature is off, the role isn't included in `valid_roles`, so the metaprogramming doesn't create the `billing_administrator?` method. Without it, I was starting to write some awkward conditions in `ability.rb`. It seemed cleaner to go this way and have the flag check in fewer places.